docs_source_files/content/worst_practices/performance_testing.en.md

```diff
@@ -38,11 +38,11 @@ you will need to be able to analyse overall performance
 independent of environment differences,
 identify poor code practices,
 breakdown of performance of individual resources
-(i.e. CSS or JavaScript)
+(i.e. CSS or JavaScript),
 in order to know what to improve.
 There are performance testing tools available
 that can do this job already,
-and which provide reporting and analysis
-which can even make improvement suggestions.
+that provide reporting and analysis,
+and can even make improvement suggestions.
 
-Example (open source) packages to use are: JMeter ?
+Example (open source) packages to use are: [JMeter](//jmeter.apache.org/)
```

Comment:
Great! Could you please add the same link to the other translations? To keep them uniform.

Reply:
Updated.